starter-code/public/scripts/article.js

```diff
@@ -60,17 +61,40 @@ Article.fetchAll = callback => {
 
 // TODO: Chain together a `map` and a `reduce` call to get a rough count of all words in all articles.
 Article.numWordsAll = () => {
-  return Article.all.map().reduce()
-};
+    return Article.all.map((articles) => {
+      var wordCount =  articles.body.split(' ');
```

Comment:
Also not a point loss, but this could also be a one line return statement, ' return article.body.split(' ').length;'. No need to waste any memory space for variables we wont necessarily reference.